tests/Core/Components/Drag/FluentDragTests.razor

```diff
@@ -0,0 +1,48 @@
+﻿@using Microsoft.FluentUI.AspNetCore.Components.Tests.Extensions
+@using Microsoft.FluentUI.AspNetCore.Components.Utilities
+@using Xunit;
+@inherits Bunit.TestContext
+
+@code {
+	public FluentDragTests()
```

Comment:
You will probably need to add additional unit tests to achieve at least 98% code coverage (100% if possible).

Reply:
Yea I know. Could you help me out by providing a testcase for the Draggable parameter? See: https://github.com/microsoft/fluentui-blazor/pull/4211#issuecomment-3371351243